sdk/storage/Azure.Management.Storage/tests/Helpers/StorageManagementTestUtilities.cs

```diff
@@ -0,0 +1,290 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.Management.Resources;
+using Azure.Management.Resources.Models;
+using Azure.Management.Storage.Models;
+using NUnit.Framework;
+using Sku = Azure.Management.Storage.Models.Sku;
+
+namespace Azure.Management.Storage.Tests.Helpers
+{
+    public static class StorageManagementTestUtilities
+    {
+        public static bool IsTestTenant = false;
+        private static HttpClientHandler Handler = null;
+
+        // These should be filled in only if test tenant is true
+#if DNX451
+        private static string certName = null;
+        private static string certPassword = null;
+#endif
+        private const string testSubscription = null;
+        //private static Uri testUri = null;
+
+        // These are used to create default accounts
+        public static string DefaultLocation = IsTestTenant ? null : "eastus2(stage)";
+        public static string DefaultRGLocation = IsTestTenant ? null : "eastus2";
+        public static Sku DefaultSkuNameStandardGRS = new Sku(SkuName.StandardGRS);
+        public static Kind DefaultKindStorage = Kind.Storage;
+
+        public static Dictionary<string, string> DefaultTags = new Dictionary<string, string>
+        {
+            {"key1","value1"},
+            {"key2","value2"}
+        };
+
+        private static HttpClientHandler GetHandler()
+        {
+#if DNX451
+            if (Handler == null)
+            {
+                //talk to yugangw-msft, if the code doesn't work under dnx451 (same with net451)
+                X509Certificate2 cert = new X509Certificate2(certName, certPassword);
+                Handler = new System.Net.Http.WebRequestHandler();
+                ((WebRequestHandler)Handler).ClientCertificates.Add(cert);
+                ServicePointManager.ServerCertificateValidationCallback += (sender, certificate, chain, sslPolicyErrors) => { return true; };
+            }
+#endif
+            return Handler;
+        }
+
+        public static StorageAccountCreateParameters GetDefaultStorageAccountParameters(Sku Sku = null, Kind? Kind = null, string Location = null)
```

Comment:
parameter name should be always camel style

Reply:
Update parameter style.